Private/Test-IsRSATInstalled.ps1

```diff
@@ -0,0 +1,15 @@
+﻿function Test-IsRSATInstalled {
+    <#
+    .SYNOPSIS
+        Tests if the RSAT AD PowerShell module is installed.
+    .DESCRIPTION
+        This function returns True if the RSAT AD PowerShell module is installed or False if not.
+    .EXAMPLE
+        Test-IsElevated
+    #>
+    if (-not(Get-Module -Name 'ActiveDirectory' -ListAvailable)) {
```

Comment:
The `-not` condition here flips your if/true logic. Just need to either remove `-net` or swap your `$true` and `$false` outputs. :)

Reply:
I agree with Sam here. Instead of having a `-not` here and a `-not` in the public `Invoke-Locksmith` function:
<img width="381" alt="image" src="https://github.com/TrimarcJake/Locksmith/assets/93942157/fde6d6bb-1f85-4da6-a960-71c80ad4e1df">
I'd rather both `-not`s be removed. It makes the logic easier to understand.